script/helperFunctions.sh

```diff
@@ -1307,6 +1307,63 @@ function getBytecodeFromArtifact() {
   fi
 }
 
+function addPeripheryToDexsJson() {
+  echo "[info] now adding all contracts listed in WHITELIST_PERIPHERY (config.sh) to config/dexs.json"
```

Comment:
Maybe better to have this in a config that is version controlled? Things in config.sh can be missed by mistake as it will differ from machine to machine.